shell/src/main/java/alluxio/cli/fsadmin/report/CapacityCommand.java

```diff
@@ -126,14 +126,16 @@ private void collectWorkerInfo(WorkerInfoOptions options) throws IOException {
             value + mSumUsedBytesOnTierMap.getOrDefault(tier, 0L));
       }
 
-      mStringBuilder.append(String.format("%-17s %-13s %-16s %-13s %-13s %-13s%n",
+      mStringBuilder.append(String.format("%-16s %-16s %-13s %-16s %-13s %-13s %-13s%n",
           workerInfo.getAddress().getHost(),
+          workerInfo.getLastContactSec(),
           "Capacity",
           FormatUtils.getSizeFromBytes(capacityBytes),
           FormatUtils.getSizeFromBytes(totalBytesOnTiers.getOrDefault("MEM", 0L)),
           FormatUtils.getSizeFromBytes(totalBytesOnTiers.getOrDefault("SSD", 0L)),
           FormatUtils.getSizeFromBytes(totalBytesOnTiers.getOrDefault("HDD", 0L))));
-      mStringBuilder.append(String.format("%-17s %-13s %-16s %-13s %-13s %-13s%n",
+      mStringBuilder.append(String.format("%-16s %-16s %-13s %-16s %-13s %-13s %-13s%n",
```

Comment:
factor out the format to a variable?

Reply:
change to 
private static final String WORKER_INFO_FORMAT = "%-16s %-16s %-13s %-16s %-13s %-13s %-13s%n";